### PR TITLE
Done (solved) issue to add games section

### PR DESCRIPTION
**Pull request by abishekvashok**
Solved issue to add games section as part of google code-in competition(task by fossasia) admins there are no conflicts now and there are titles and also no bugs so please do the needy and accept my task(you know time is money)

### CODE REVIEW

**Comment by prwatch** [comment]
DEMO: http://Abhi2424shek.github.io/gci15.fossasia.org

File: index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+﻿---
```
Comment (roonyh) [inline index.html:1]: Let those things be. They are important to Jekyll

**Comment by abishekvashok** [comment]
@roonyh @sudheesh001 @mariobehling @nikunjness  please help me out ok :) [by accepting the task, I've done all that you wanted me to :(  :+1:

**Comment by roonyh** [comment]
Try opening http://abhi2424shek.github.io/gci15.fossasia.org Doesn't look ok. And left right arrows from your previous PR are not there.

Also there already is a section for Games. Can you add the left right arrows to that section?

**Comment by abishekvashok** [comment]
I would resolve the issue by adding the games section with arrows that works only with the games section and the gallery section. And here would be  no side effects. I would make a new pr instead making a commit qgain to resolve the conflicts and squashing is not helping me either so please wait till I come from school in the evening i will solve it by evening now no time i will have to go to school ok :)

File: index.html (hunk 1 above)
Comment (abishekvashok) [inline index.html:1]: Sorry for that i never did it willfully and i new the site was build using jekyll :)

**Comment by roonyh** [comment]
Sure. Thanks!

**Pull request closed without merging**